Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+elifePipeline {
+    def commit
+    def commitShort
+    def schemaVersion
```

Comment:
variables not used outside of the `Checkout` stage can be defined inside the `Checkout` stage

Reply:
Think I've resolved this now